docs/parsers/CanvasPathGeometry.md

```diff
@@ -0,0 +1,223 @@
+
+While the `CanvasGeometry` class provides various static methods to create predefined shapes like Circle, Ellipse, Rectangle, RoundedRectangle, the `CanvasPathBuilder` class provides several methods to create freeform CanvasGeometry objects.
+
+Creation of a complex freeform geometric shape may involve invoking of several CanvasPathBuilder commands. For example, the following code shows how to create a triangle geometry using CanvasPathBuilder
+
+```cs
+CanvasPathBuilder pathBuilder = new CanvasPathBuilder(device);
+pathBuilder.BeginFigure(1, 1);
+pathBuilder.AddLine(300, 300);
+pathBuilder.AddLine(1, 300);
+pathBuilder.EndFigure(CanvasFigureLoop.Closed);
+CanvasGeometry triangleGeometry = CanvasGeometry.CreatePath(pathBuilder);
+```
+
+**Win2d Path Mini Language** is a powerful and sophisticated language based on the [SVG Path language](https://www.w3.org/TR/SVG11/paths.html) specification. It facilitates specifying complex geometries, color, brushes, strokes and stroke styles in a more compact manner.
+
+[More details about Win2d Path Mini Language](https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/parsers/Win2d_Path_Mini_Language.md)
+
+The `CanvasPathGeometry` class provides several static methods that parse the Win2d Mini Language and instantiate the complex geometries, color, brushes, strokes and stroke styles. This class is available in the `Microsoft.Toolkit.Uwp.UI.Media.Geometry` namespace.
+
+Using Win2d Path Mini Language, the geometry in above example can be created in the following way
+
+```cs
+string pathData = “M 1 1 300 300 1 300 Z”;
+CanvasGeometry triangleGeometry = CanvasPathGeometry.CreateGeometry(device, pathData);
+```
+
+> [!div class="nextstepaction"][try it in the sample app](uwpct://Helpers?sample=CanvasPathGeometryPage)
+
+## Sample Project
+
+<!-- Link to the sample page in the Windows Community Toolkit Sample App -->
+
+[CanvasPathGeometry Sample Page Source](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/CanvasPathGeometry/CanvasPathGeometryPage.xaml.cs). You can [see this in action](uwpct://Helpers?sample=CanvasPathGeometryPage) in [Windows Community Toolkit Sample App](http://aka.ms/uwptoolkitapp).
+
+## Requirements
+
+| Device family | Universal, MinVersion or higher                                                                  |
+| ------------- | ------------------------------------------------------------------------------------------------ |
+| Namespace     | Microsoft.Toolkit.Uwp.UI.Media.Geometry                                                          |
+| NuGet package | [Microsoft.Toolkit.Uwp.UI.Media](https://www.nuget.org/packages/Microsoft.Toolkit.Uwp.UI.Media/) |
+
+## API
+
+- [CanvasPathGeometry](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI.Media/Geometry/CanvasPathGeometry.cs)
```

Comment:
Should keep these things at the bottom, and probably just move all the examples up under an 'Examples' section.